src/components/ContentBlocks/BlockRichText/BlockRichText.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from 'react';
+
+import classnames from 'classnames';
+
+import { convertToHtml } from '../../../helpers/convertToHtml';
+import { DefaultProps } from '../../../types';
+import { Container } from '../../Container/Container';
+import { Column, GridSize } from '../../Grid/Column/Column';
+import { Grid } from '../../Grid/Grid';
+
+export interface BlockRichTextProps extends DefaultProps {
+	content: string | string[];
+}
+
+export const BlockRichText: FunctionComponent<BlockRichTextProps> = ({
+	className,
+	content = [''],
+}) => (
+	<Container className={classnames(className, 'o-container-vertical-rich-text')} mode="vertical">
```

Comment:
Ik zou op de blocks niets van containers zetten aangezien we die padding zelf gaan laten instellen.
De width wordt dan weer op page niveau voorzien

Reply:
Ik ga akkoord tbh.